src/main/java/randoop/main/Minimize.java

```diff
@@ -0,0 +1,1374 @@
+        "A minimized JUnit test suite (as one Java file) named \"InputFileMinimized.java\" if \"InputFile.java\" were the name of the input file.",
+        "java -ea -cp bin:randoop-all-3.1.1.jar randoop.main.Main minimize --suitepath=~/RandoopTests/src/ErrorTestLang.java --suiteclasspath=~/RandoopTests/commons-lang3-3.5.jar --testsuitetimeout=30",
+        new Options(Minimize.class));
+  }
+
+  private static final String PATH_SEPARATOR = System.getProperty("path.separator");
+  private static final String SYSTEM_CLASS_PATH = System.getProperty("java.class.path");
+
+  // The suffix to postpend onto the name of the minimized file and class.
+  private static final String SUFFIX = "Minimized";
+
+  /**
+   * Check that the required parameters have been specified by the command-line options and then
+   * call the mainMinimize method.
+   *
+   * @param args parameters, specified in command-line style, for the input file, the classpath, the
+   *     timeout value, and the verbose flag
+   * @return true if the command was handled successfully
+   * @throws RandoopTextuiException thrown if incorrect arguments are passed
+   */
+  @Override
+  public boolean handle(String[] args) throws RandoopTextuiException {
+    try {
+      String[] nonargs = foptions.parse(args);
+      if (nonargs.length > 0) {
+        throw new Options.ArgException("Unrecognized arguments: " + Arrays.toString(nonargs));
+      }
+    } catch (Options.ArgException ae) {
+      usage("while parsing command-line arguments: %s", ae.getMessage());
+    }
+
+    if (Minimize.suitepath == null) {
+      System.out.println("You must specify an input file path.");
+      System.out.println("Use the --suitepath option.");
+      System.exit(1);
+    }
+
+    // Check that the input file is a Java file.
+    if (!FilenameUtils.getExtension(Minimize.suitepath).equals("java")) {
+      System.out.println("The input file must be a Java file.");
+      System.exit(1);
+    }
+
+    if (Minimize.testsuitetimeout <= 0) {
+      System.out.println("You must specify a positive timeout value.");
+      System.exit(1);
+    }
+
+    // Call the main minimize method.
+    return mainMinimize(suitepath, suiteclasspath, testsuitetimeout, verboseminimizer);
+  }
+
+  /**
+   * Minimize the input test file.
+   *
+   * <p>Given an input Java file, minimization produces an output file that is as small as possible
+   * (as few lines of code as possible) and that fails the same way:
+   *
+   * <ol>
+   *   <li>Same failing assertions as in the original input test suite.
+   *   <li>Same stacktrace produced by failing assertions.
+   * </ol>
+   *
+   * <p>The original input Java file will be compiled and run once. The "expected output" is a map
+   * from test method name to failure stack trace. A method is included in the map only if the
+   * method contains a failing assertion. Thus, the expected output of a test suite with no failing
```

Comment:
I think the problem with my comment here is that I'm intending for "expected output" to mean the standard output obtained from running the input test suite.

I have changed the comment to:
`The original input Java file will be compiled and run once. The "expected output" derived from the standard output from running the input file is a map from test method name to failure stack trace. A method is included in the map only if the method contains a failing assertion. Thus, the "expected output" of running a test suite with no failing tests will be an empty map. The "expected output" will be used in subsequent runs of the modified test suite to determine whether or not the test suite still fails in the same way.`